CHANGES.rst

```diff
@@ -13,6 +13,9 @@ https://github.com/zopefoundation/Zope/blob/4.x/CHANGES.rst
 
 - Update to newest compatible versions of dependencies.
 
+- Use string representation of response data that has no ``__bytes__``
```

Comment:
Suggestion: "Use intermediate ``str`` representation for complex response data (i.e. not `str` or bytes like)